egs/thchs30/s5/path.sh

```diff
@@ -0,0 +1,18 @@
+#Change the following settings to match your computing env,
+#so that appropriate (CPU/GPU) Kaldi commands are visible
+
+export KALDI_ROOT=/work3/zxw/kaldi-2016-1-21
```

Comment:
yes, as Yenda says, the normal setting is something like KALDI_ROOT=../../..
Bear in mind that the point with all these scripts is to make it easy for it to work on *someone else's* system.  Whether it works easily on your system is a very secondary consideration-- for you, you know how your system works and you can easily change things.  Someone who downloads your recipe doesn't need extra hassle.